Source/WebCore/bindings/scripts/preprocess-idls.pl

```diff
@@ -501,6 +501,15 @@ sub GenerateConstructorAttributes
       $extendedAttributes->{"Conditional"} = $existingConditional;
     }
 
+    if ($globalContext eq "ShadowRealm" && $extendedAttributes->{"Exposed"} eq "*") {
```

Comment:
Per @Ms2ger, since ShadowRealm is available in Worklets, hiding apis exposed in ShadowRealm from worklets doesn't really prevent their use within a Worklet, so I think we just leave it as-is